mobile/src/libs/components/card/card.tsx

```diff
@@ -16,27 +16,23 @@ import { styles } from './styles';
 
 type Properties = {
   title: string;
-  id?: string;
+
   image?: ImageSourcePropType;
   iconName?: IconName;
   iconColor?: string;
-  onPress: (title: string, id?: string) => void;
+  onPress?: () => void;
+  id?: number | string;
```

Comment:
Why two types?

Reply:
@v0ldemar01 because in my task i used id as number and someone created id as a string, so not to ruin code with string id  i created this